library/src/dotty/DottyPredef.scala

```diff
@@ -37,6 +37,11 @@ object DottyPredef {
     case ev: ValueOf[T] => ev.value
   }
 
+  /** Summon an given value of type `T`. Usually, the argument is not passed explicitly.
+   *
+   *  @tparam T the type of the value to be summoned
+   *  @return the given value typed as the given parameter
```

Comment:
The terminology may be confusing here.

```suggestion
   *  @return the given value typed as the provided type parameter
```